extensions/amp-viewer-integration/0.1/amp-viewer-integration.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {Messaging, WindowPortEmulator, parseMessage} from './messaging';
+import {Messaging, WindowPortEmulator, parseMessage} from 'messaging';
```

Comment:
Can you explain how "messaging" name here is getting resolved? Also, what will this look like when imported from the "amp-viewer" repo?